README.md

```diff
@@ -51,7 +51,9 @@ container recreates.
 DNS records to configure can be found in the container log or by running
 `docker exec <container> cat /etc/opendkim/keys/*/*.txt`.
 
-Default selector is "mail" but can be changed using the syntax `domain.tld=abc`.
+The default selector is "mail", but can be changed to "<selector>" by supplying
+the file `/etc/opendkim/keys/<domain>/<selector>.private` and the environment
+variable `OPENDKIM_DOMAINS=<domain>=<selector>`.
```

Comment:
Make it clear that the file is optional and will be generated if does not exist?

Reply:
This is a good point -- thanks for the feedback. I have hopefully clarified sufficiently in my new commit. I had to reword a little to avoid repeating too much information (which I think would have made the instructions less clear). How does it look now?